tests/test_signal.py

```diff
@@ -78,15 +75,12 @@ def test_domain_setter_time_when_time():
     assert signal.domain == domain
 
 
-def test_domain_setter_time_when_freq(sine_stub):
-    signal = Signal(
-        sine_stub.freq, sine_stub.sampling_rate, domain='freq',
-        fft_norm=sine_stub.fft_norm)
+def test_domain_setter_time_when_freq():
+    signal = Signal([1, 2, 3, 4], 44100, domain='time', fft_norm='rms')
```

Comment:
```suggestion
    signal = Signal([1, 2, 3, 4], 44100, domain='freq', fft_norm='rms')
```
If I get it right, the domain of the Signal defined for this test should have the 'freq' domain in order to test the setter below.